src/prognostic_equations/gm_sgs_closures.jl

```diff
@@ -61,7 +61,7 @@ Modifies `ᶜmixing_length` in place. Also modifies fields in `p.precomputed`
 (like `ᶜlinear_buoygrad`, `ᶜstrain_rate_norm`) and uses `p.scratch` for
 intermediate calculations.
 """
-NVTX.@annotate function compute_gm_mixing_length!(ᶜmixing_length, Y, p)
+NVTX.@annotate function compute_gm_mixing_length!(Y, p)
```

Comment:
```suggestion
NVTX.@annotate function compute_gm_mixing_length(Y, p)
```
This function no longer modifies the input argument, so shouldn't the exclamation point be dropped?